docs/content/latest/troubleshoot/cluster/replace_master.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Replace a failed YB-Master
+headerTitle: Replace a failed YB-Master
+linkTitle: Replace a failed YB-Master
+description: Steps to replace a failed YB-Master in a YugabyteDB cluster.
+aliases:
+  - /troubleshoot/cluster/replace-master/
+  - /latest/troubleshoot/cluster/replace-master/
+menu:
+  latest:
+    identifier: replace-failed-master
+    parent: troubleshoot-cluster
+    weight: 831
+isTocNested: true
+showAsideToc: true
+---
+
+To replace a failed YB-Master server in a YugabyteDB cluster, follow these steps:
+
+For the steps below, the examples use the following scenario:
+
+- The cluster includes three `yb-master` servers: `M1`, `M2`, `M3`.
+- YB-Master server `M1` failed and needs to be replaced.
+- A new YB-Master server (`M4`) will replace `M1`.
+- The default master RPC port is `7100`
+
+
+1. Start the new (replacement) YB-Master server in standby mode. 
+
+To start `yb-master` in standby mode, set the `--master_addresses` flag to an empty string (`""`).
+
+```sh
+./bin/yb-master --master_addresses=""
+```
+
+2. Add the new YB-Master server into the existing cluster.
+
+To add the new YB-Master server, run the [`yb-admin change_master_config ADD_SERVER`](../../admin/cli/yb-admin/#change-master-config) command.
+
+```sh
+./bin/yb-admin -master_addresses M1:7100,M2:7100,M3:7100 change_master_config ADD_SERVER M4 7100
+```
+
+3. Remove the old YB-Master server from the cluster.
+
+To remove the failed YB-Master server from the cluster, use the [`yb-admin change_master_config REMOVE_SERVER`](../../admin/yb-admin/#change-master-config`) command.
+
+```sh
+./yb-admin -master_addresses M1:7100,M2:7100,M3:7100,M4:7100 change_master_config REMOVE_SERVER M1 7100
+```
+{{< note title="Note" >}}
+
+Make sure to specify all YB-Master addresses which will prevent the new YB-Master server from unexpectedly becoming the leader.
```

Comment:
hmm, the end part of this comment does not make sense...you need to specify M4, not to prevent it from becoming the leader (this command cannot affect that), but instead, to make sure that, if it DID become the leader in the meantime, then yb-admin can find it

Reply:
Check change here https://github.com/yugabyte/yugabyte-db/pull/4791/commits/67996cc5a0d936e7b8b6d6fe45aa43b30e5f4933 ?